BunqSdk/Exception/ExceptionFactory.cs

```diff
@@ -7,18 +7,18 @@ public class ExceptionFactory
         /// <summary>
         /// HTTP error response codes constants.
         /// </summary>
-        private const int HTTP_RESPONSE_CODE_BAD_REQUEST = 400;
-        private const int HTTP_RESPONSE_CODE_UNAUTHORIZED = 401;
-        private const int HTTP_RESPONSE_CODE_FORBIDDEN = 403;
-        private const int HTTP_RESPONSE_CODE_NOT_FOUND = 404;
-        private const int HTTP_RESPONSE_CODE_METHOD_NOT_ALLOWED = 405;
-        private const int HTTP_RESPONSE_CODE_TOO_MANY_REQUESTS = 429;
-        private const int HTTP_RESPONSE_CODE_INTERNAL_SERVER_ERROR = 500;
+        private const int HttpResponseCodeBadRequest = 400;
+        private const int HttpResponseCodeUnauthorized = 401;
+        private const int HttpResponseCodeForbidden = 403;
+        private const int HttpResponseCodeNotFound = 404;
+        private const int HttpResponseCodeMethodNotAllowed = 405;
+        private const int HttpResponseCodeTooManyRequests = 429;
+        private const int HttpResponseCodeInternalServerError = 500;
         
         /// <summary>
         /// Glue to concatenate the error messages.
         /// </summary>
-        private const string GLUE_ERROR_MESSAGES = "\n";
+        private const string GlueErrorMessages = "\n";
```

Comment:
ErrorMessageGlue

Reply:
`GlueAllErrorMessage` Group by `Glue`

Reply:
I don't like the `Glue` too much. Why don't we use `Separator` or sth?

Reply:
Hmm well `Separator` can also be used just fine 👍